src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -147,6 +160,21 @@ class ClosedLoans extends React.Component {
     return contributorsList;
   }
 
+  buildRecords(recordsLoaded) {
```

Comment:
Seems like the `buildRecords` method is exactly the same as the one in `OpenLoans.js` file. Could you please extract in order and use in both files in order to avoid duplication?